packages/app/src/vis-packs/core/line/LineVisContainer.tsx

```diff
@@ -39,14 +40,15 @@ function LineVisContainer(props: VisContainerProps) {
       >
         <ValueFetcher
           dataset={entity}
-          selection={autoScale ? getSliceSelection(dimMapping) : undefined}
+          selection={selection}
           render={(value) => (
             <MappedLineVis
+              dataset={entity}
+              selection={selection}
```

Comment:
Given how often `dataset` and `selection` come together, it might be tempting to merge them in a single object

Reply:
Interesting thought, but do you mind if we keep thinking about this one for a bit? I feel like there might be a broader refactor to be done after all of the recent toolbar work.